### DISCUSSION TITLE
Javascript

### DISCUSSION THREAD

**Discussion by SabaNadeemUOS** [SabaNadeemUOS/WanderLust#39] [Ideas]
Lets do our javascript discussions here

**Comment by SabaNadeemUOS** [comment]
Which parts of the website do you guys want to take for the javascript?
@asmasulaibeekh @Lamyaalmheiri @sarahalhammadi @shaikhaalz

**Reply by sarahalhammadi** [reply]
i will do the focus on the form elements

**Reply by shaikhaalz** [reply]
I will try to make a popup with "Thank you for the feedback".

**Comment by SabaNadeemUOS** [comment]
There are a lot of codes available online and the most best website to look at would be w3schools so you guys can look there for ideas

**Comment by Mouza-Ayman-Alsharhan** [comment]
i will do a javascript for a button that is when clicked it will scroll up to the beginning of the page. 
Look at the attached video

https://user-images.githubusercontent.com/113240180/202859666-3dbe1545-a21c-4dca-9be2-cdbef6b62d7f.mp4

**Comment by SabaNadeemUOS** [comment]
I will do a javascript where you can change the background color of the page by clicking on the available colors
Here is the video attached
Suggest some colors for me to add guys!
And i am facing some trouble in finding a good place to adjust it.Where ever i put it it always pushes the other elements around it.
https://uoshj-my.sharepoint.com/:v:/g/personal/u20102142_sharjah_ac_ae/EUnG5zl0WntHtdZ0dzlWoFgBSdST2eHNckyS7-e1s29QiA?e=blfHrD

**Reply by Mouza-Ayman-Alsharhan** [reply]
I liked that, 
I think light blue color from the same hue of our website would look good.

for adjusting the place problem, what did you choose for the 'position' attribute?

**Reply by SabaNadeemUOS** [reply]
i don't have a position attribute
I am talking about the overall placement 
i can't keep it next to the logo because the logo gets pushed down
I can't keep it on the sides because it pushes the text

**Reply by Mouza-Ayman-Alsharhan** [reply]
Saba,
Can you try adding the following attributes:

position:fixed;
  bottom: 20px;
  right: 30px;

**Reply by SabaNadeemUOS** [reply]
the code is actually just a table with no borders
What will your code do exaclty?

**Reply by Mouza-Ayman-Alsharhan** [reply]
the attributes that I just mention will fix the table so the right-top corner of the page, and if you even if you scroll down it won't disappear.

**Reply by SabaNadeemUOS** [reply]
ohh i get it now
Thank you!
